cmd/util/cmd/read-badger/cmd/collections.go

```diff
@@ -34,19 +36,25 @@ var collectionsCmd = &cobra.Command{
 			}
 
 			log.Info().Msgf("getting collection by id: %v", collectionID)
-			collection, err := storages.Collections.ByID(collectionID)
-			if err != nil {
-				log.Error().Err(err).Msgf("could not get collection with id: %v", collectionID)
+
+			// get only the light collection if specified
+			if flagLightCollection {
+				light, err := storages.Collections.LightByID(collectionID)
+				if err != nil {
+					log.Error().Err(err).Msgf("could not get collection with id: %v", collectionID)
+					return
+				}
+				common.PrettyPrintEntity(light)
 				return
 			}
 
-			common.PrettyPrintEntity(collection)
-			// print each transaction id
-
-			for i, tx := range collection.Transactions {
```

Comment:
This command actually did print out the transaction IDs in a separate log which I somehow missed during the fire so this isn't strictly necessary. I still think it's worth putting that fact in the flag descriptions.